src/routes/settings/components/VersionPressable.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import { getVersion } from 'react-native-device-info'
+import styled from 'styled-components/native'
+
+import { ContentSecondary } from '../../../components/text/Content'
+import labels from '../../../constants/labels.json'
+
+const Version = styled.Pressable`
+  position: absolute;
+  bottom: 0;
+  padding: ${props => props.theme.spacings.xl} ${props => props.theme.spacings.md};
+`
+
+interface PropsType {
+  setVisible: () => void
+}
+
+export const CLICKS_TO_THROW_SENTRY_ERROR = 10
```

Comment:
Since this component is not sentry specific
```suggestion
export const CLICK_THRESHOLD = 10
```